packages/jest-repl/src/cli/repl.ts

```diff
@@ -24,7 +24,7 @@ const evalCommand: repl.REPLEval = (
 ) => {
   let result;
   try {
-    if (transformer) {
+    if (transformer && transformer.process) {
```

Comment:
this probably should support async transforms as well, but leaving a `TODO` comment is enough I think. But we should probably throw instead of silently ignoring